test/manifests/setup.yaml

```diff
@@ -114,6 +122,14 @@ spec:
     image: public.ecr.aws/karpenter/tools:latest
     workingDir: $(workspaces.ws.path)
     script: |
+      # Use more cpu/memory when running scale tests
+      cpuCount=1
+      memoryCount="1Gi"
+      if [[ "$(params.test-filter)" == "Scale" ]]; then
+        cpuCount=4
+        memoryCount="4Gi"
+      fi
+      
```

Comment:
Is this value tuned for 200 node scale tests? Thoughts on fully parameterizing this? Since we'll probably be testing many different dimensions, probably helps to have this knob in too.

Reply:
That seems like a change that we can make down-the-line. I think using these values as an upper-bound is fine for now